reader/processor/processor.go

```diff
@@ -12,17 +12,23 @@ import (
 	"unicode/utf8"
 
 	"miniflux.app/config"
+	"miniflux.app/http/client"
 	"miniflux.app/logger"
 	"miniflux.app/metric"
 	"miniflux.app/model"
+	"miniflux.app/reader/browser"
 	"miniflux.app/reader/rewrite"
 	"miniflux.app/reader/sanitizer"
 	"miniflux.app/reader/scraper"
 	"miniflux.app/storage"
 
+	"github.com/PuerkitoBio/goquery"
 	"github.com/rylans/getlang"
+	"github.com/senseyeio/duration"
```

Comment:
Do we really need to introduce a new dependency just to parse the duration?

It's just one function: https://github.com/senseyeio/duration/blob/master/duration.go#L30-L72